src/lib.rs

```diff
@@ -72,7 +69,9 @@ mod fp12;
 mod fp6;
 
 // The BLS parameter x for BLS12-381 is -0xd201000000010000
+#[cfg(feature = "std")]
 const BLS_X: u64 = 0xd201000000010000;
+#[cfg(feature = "std")]
```

Comment:
Why these constants are only `std` avaliable?

Reply:
Fixed. was part of a local test and this feature constaint is indeed not required

Reply:
KK perfect!